src/main/java/com/blibli/future/controller/AttendanceController.java

```diff
@@ -101,38 +102,37 @@ public ResponseEntity employeeTappingUpdate(@RequestParam("type") String type, @
         return new ResponseEntity(employeeTapGetted, HttpStatus.BAD_REQUEST);
```

Comment:
I'm sorry this one really bothers me.
The past participle of "get" is "got/gotten" not "getted" 
"getted" is a non standard form [https://en.wiktionary.org/wiki/getted](url)